src/app/components/lightsaber.js

```diff
@@ -4,15 +4,18 @@ class Lightsaber {
         this.render(id);
     }
 
-    contentRender(id) {
-        // rozbijmy to na dwie części
+    handleRender(id) {
         const container = document.querySelector(`#${id}`);
         container.classList.add('saber');
         const pictureSrc = '../static/assets/ui/LightsaberHandle.png';
         const picture = document.createElement('img');
         picture.classList.add('saber__handle');
         picture.setAttribute('src', pictureSrc);
         container.appendChild(picture);
+    }
+
+    saberRender(id) {
+        const container = document.querySelector(`#${id}`);
```

Comment:
Przeniśmy to do render i przekażmy wtedy do obu funkcji zamiast id, dzieki temu nie będzie trzeba pisać tego samego 2 razy

Reply:
o widzisz! dzięki :)